src/string-builder.js

```diff
@@ -27,4 +27,20 @@ StringBuilder.prototype.cat = function(){
 
 StringBuilder.prototype.string = function() {
     return this.buffer.join('');
+};
+
+StringBuilder.prototype.rep = function(...args) {
+    var buffer = this.buffer;
+    var howManyTimes = args.pop();
+
+    if (typeof howManyTimes !== 'number') {
+        throw new TypeError('Expected a number');
+    }
+
+    let callback = element => this.cat(element);
+    for (let i=0; i < howManyTimes; i++) {
```

Comment:
What about `this.cat(args)` or `this.cat.apply(this, args)`? will reduce the complexity of using the `callback`